CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.X.X] - 2019-07-17
+- New Features
+   - Added new velocity format options to utils.coords.scale_units
+   - Improved failure messages for utils.coords.scale_units
+- Bug fix
+   - Fixed implementation of utils routines in model_utils and jro_isr
+
 ## [2.0.0] - 2019-07-??
```

Comment:
We should update with the actual release date of 2.0.0

Reply:
My bad, sorry about that.

Reply:
fixed.